schemas/data.schema.json

```diff
@@ -273,117 +157,146 @@
-              "$ref": "#/definitions/BrowserData"
+              "type": "string"
             },
             "chrome_android": {
-              "$ref": "#/definitions/BrowserData"
+              "type": "string"
             },
             "edge": {
-              "$ref": "#/definitions/BrowserData"
+              "type": "string"
             },
             "firefox": {
-              "$ref": "#/definitions/BrowserData"
+              "type": "string"
             },
             "firefox_android": {
-              "$ref": "#/definitions/BrowserData"
+              "type": "string"
             },
             "safari": {
-              "$ref": "#/definitions/BrowserData"
+              "type": "string"
             },
             "safari_ios": {
-              "$ref": "#/definitions/BrowserData"
+              "type": "string"
             }
           },
-          "required": [
-            "chrome",
-            "chrome_android",
-            "edge",
-            "firefox",
-            "firefox_android",
-            "safari",
-            "safari_ios"
-          ],
-          "type": "object"
+          "additionalProperties": false
+        }
+      },
+      "required": ["baseline", "support"]
+    },
+    "StatusHeadline": {
+      "type": "object",
+      "allOf": [
+        { "$ref": "#/definitions/Status" },
+        {
+          "type": "object",
+          "properties": {
+            "by_compat_key": {
+              "description": "Statuses for each key in the feature's compat_features list, if applicable. Not available to the npm release of web-features.",
+              "type": "object",
+              "additionalProperties": {
+                "$ref": "#/definitions/Status"
+              }
+            }
+          }
+        }
+      ]
+    },
+    "Release": {
+      "description": "Browser release information",
+      "type": "object",
+      "properties": {
+        "version": {
+          "description": "The version string, as in \"10\" or \"17.1\"",
+          "type": "string"
         },
-        "features": {
-          "additionalProperties": {
-            "$ref": "#/definitions/FeatureData"
-          },
-          "description": "Feature identifiers and data",
-          "type": "object"
+        "date": {
+          "description": " The release date, as in \"2023-12-11\"",
+          "type": "string"
+        }
+      },
+      "required": ["version", "date"],
+      "additionalProperties": false
+    },
+    "SnapshotData": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "Short name",
+          "type": "string"
         },
-        "groups": {
-          "additionalProperties": {
-            "$ref": "#/definitions/GroupData"
-          },
-          "description": "Group identifiers and data",
-          "type": "object"
+        "spec": {
+          "description": "Specification",
+          "$ref": "#/definitions/URL"
+        }
+      }
+    },
```

Comment:
FYI: name and spec were previously marked as required. Was it intentional?

Reply:
Fixed in b2b6762.